tests/engines/registries/acceptance/draft/draft-test.ts

```diff
@@ -422,6 +424,75 @@ module('Registries | Acceptance | draft form', hooks => {
         assert.dom('[data-test-page-label]').containsText('This is the second page');
     });
 
+    test('it shows failed saves/deletes', async function(this: DraftFormTestContext, assert) {
+        server.namespace = '/v2';
+        server.patch('/draft_registrations/:draftId', () => ({
+            errors: [{ detail: 'Fail to save draft' }],
+        }), 400);
+        server.del('/draft_registrations/:draftId', () => ({
+            errors: [{ detail: 'Fail to delete draft' }],
+        }), 403);
+        const initiator = server.create('user', 'loggedIn');
+        const registrationSchema = server.schema.registrationSchemas.find('testSchema');
+        const registration = server.create(
+            'draft-registration', {
+                registrationSchema,
+                initiator,
+                branchedFrom: this.branchedFrom,
+            },
+        );
+
+        // Fail metadata save
+        await visit(`/registries/drafts/${registration.id}/`);
+        await triggerKeyEvent('[data-test-metadata-title] input', 'keyup', 32);
+        assert.dom('#toast-container', document as any).containsText(
+            t('registries.drafts.draft.metadata.failed_auto_save'),
+            'Error toast on failed metadata save',
+        );
+        // wait for toast to disappear. TODO: find a better way to do this. sinon.useFakeTimers()??
+        await timeout(5000);
+
+        // Fail form save
+        await click('[data-test-goto-next-page]');
+        await triggerKeyEvent('[data-test-text-input] input', 'keyup', 32);
+        assert.dom('#toast-container', document as any).containsText(
+            t('registries.drafts.draft.form.failed_auto_save'),
+            'Error toast on failed page save',
+        );
```

Comment:
Why don't you have to wait after this one?

Reply:
So I really don't know why, but for whatever reason, the metadata and form save toast won't show up if another toast error is on screen. The delete error toast shows up just fine, even if another toast is on screen.

Reply:
Oh, it doesn't show the new message because the toast message content is the same... 😑